### PR TITLE
feat: migrate to CRA v4

### LINKED ISSUE #16: please upgrade to jest@26+
this currently using jest@24 which causes issues (https://github.com/microsoft/create-react-app-uifabric/issues/14)
+ this creates a boilerplate project, it should make an effort to include the most maintained dependencies

### PR DESCRIPTION
**Pull request by Hotell**
Closes #16

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": true
```
Comment (Hotell) [inline .prettierrc:2]: added prettier to get formatting issues out of the way

File: package.json [hunk 2]
```diff
@@ -3,15 +3,19 @@
...
-    "@types/jest": "24.0.12",
-    "@types/node": "12",
-    "@types/react": "16.8.15",
-    "@types/react-dom": "16.8.4",
     "@fluentui/react": "^7.155.3",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-scripts": "3.4.4",
-    "typescript": "3.9.7"
+    "@testing-library/jest-dom": "^5.11.4",
```
Comment (Hotell) [inline package.json:7]: all of those are part of CRAv4 template

File: src/index.tsx [hunk 3]
```diff
@@ -2,20 +2,20 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { App } from './App';
 import { mergeStyles } from '@fluentui/react';
-import * as serviceWorker from './serviceWorker';
```
Comment (Hotell) [inline src/index.tsx:5]: service worker is no longer shipped with default CRAv4 ( pwa template needs to be used )

File: .prettierrc (hunk 1 above)
Comment (ecraig12345) [reply .prettierrc:2]: Good idea. Not super important but I wonder if we should also add these settings (used in the main repo):
```
  printWidth: 120, // or 100? 
  tabWidth: 2,
  trailingComma: 'all',
```

**Review by ecraig12345 — APPROVED** [review]
One minor suggestion but otherwise LGTM

File: .prettierrc (hunk 1 above)
Comment (Hotell) [reply .prettierrc:2]: We can adjust these things once we migrate this to official CRA template under @fluentui repo.

**Pull request merged**